src/Files.App/DataModels/AppModel.cs

```diff
@@ -109,14 +109,7 @@ private void DetectFontName()
 			var newIconsMinVersion = new Version(10, 0, 21327, 1000);
 			bool isRunningNewIconsVersion = currentVersion >= newIconsMinVersion;
 
-			if (isRunningNewIconsVersion)
-			{
-				SymbolFontFamily = new FontFamily("Segoe Fluent Icons");
-			}
-			else
-			{
-				SymbolFontFamily = new FontFamily("Segoe MDL2 Assets");
-			}
+			SymbolFontFamily = (isRunningNewIconsVersion) ? new FontFamily("Segoe Fluent Icons") : new FontFamily("Segoe MDL2 Assets");
```

Comment:
Maybe rename `isRunningNewIconsVersion` to `isWindows11`.

Reply:
Done.